app/auth/middleware.js

```diff
@@ -12,9 +12,7 @@ function processAuthResponse() {
     }
 
     try {
-      const userLocations = await userLocationService.getUserLocations(
-        grant.response.access_token
```

Comment:
Wonder if it's worth setting `grant.response.access_token` to a `const`? It's called a number of times in this method now.